index.js

```diff
@@ -7,36 +7,40 @@ const fp = require('fastify-plugin')
 const bankai = require('bankai')
 
 function assetsCompiler (fastify, opts, next) {
-  assert(opts.entryFile, 'Missing entry file!')
-  assert.ok(typeof opts.entryFile === 'string', 'entryFile must be a string')
-  if (opts.html) assert.ok(typeof opts.html === 'string', 'html must be a string')
-  if (opts.baseURL) assert.ok(typeof opts.baseURL === 'string', 'baseURL must be a string')
-  if (opts.options) assert.ok(typeof opts.options === 'object', 'options must be an object')
+  if (!opts.entry) {
+    return next(new Error('Missing entry file!'))
+  }
+  if (typeof opts.entry !== 'string') {
+    return next(new Error('entry must be a string'))
+  }
+  if (opts.html && typeof opts.html !== 'string') {
+    return next(new Error('html must be a string'))
+  }
 
   const html = !!opts.html
   const htmlPath = resolve(opts.html || '')
-  const assets = bankai(resolve(opts.entryFile || ''), opts.options || {})
+  const assets = bankai(resolve(opts.entry || ''), opts)
```

Comment:
Do we need to remove the `prefix` key from the option object before passing it to `bankai`?

Reply:
done